Client/Frontend/Browser/TabManager.swift

```diff
@@ -518,84 +518,86 @@ class TabManager: NSObject {
     }
 }
 
-extension TabManager {
+class SavedTab: NSObject, NSCoding {
```

Comment:
hmm. I'll test it a bit more. But thats exactly the crash I was getting. Moving it outside of TabManager is what fixed it.